.gitlab-ci.yml

```diff
@@ -84,5 +84,5 @@ prod_deploy:
   only:
     - tags
   script:
-    -  helm pull oci://$HARBOR/tulibraries/tupress-charts/tupress --version "0.3.6" --untar
-    - helm upgrade tupress oci://$HARBOR/tulibraries/tupress-charts/tupress --version "0.3.6" --history-max=5 --namespace=tupress-prod --values tupress/values-prod.yaml --set image.repository=$IMAGE:$CI_COMMIT_TAG
+    -  helm pull oci://$HARBOR/tulibraries/tupress-charts/tupress --version "0.3.8" --untar
```

Comment:
@cdoyle-temple I had to make some updates when working on the alerts so the latest version is 0.3.9